### PR TITLE
Fix GA blog post issue template for clarity and new assignee

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

**Comment by lauracowen** [comment]
#run-libby-bot

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ name: Open Liberty GA release blog post
...
 
 Please provide the following information the week before the GA date (to allow for review and publishing):
 
-1. Which Liberty feature does your update relate to?
+1. Which Liberty feature(s) does your update relate to?
     
    Human-readable name (eg WebSockets feature):
    
-   Feature name (eg websockets-1.0): 
+   Feature config name (eg websockets-1.0):
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/blog_post_ga_release.md:18]: `Short Liberty feature names` instead?

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md [hunk 2]
```diff
@@ -35,4 +35,4 @@ If you have previously provided this information for an Open Liberty beta blog p
 
 - Your paragraph will be included in the GA release blog post. It might be edited for style and consistency.
 - You will be asked to review a draft before publication.
-- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the GA blog post).
\ No newline at end of file
+- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the GA blog post).
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/blog_post_ga_release.md:38]: My spot the difference skills are failing me! These look the same?

**Review by tevans78 — APPROVED** [review]

**Comment by lauracowen** [comment]
#run-libby-bot

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md (hunk 2 above)
Comment (lauracowen) [reply .github/ISSUE_TEMPLATE/blog_post_ga_release.md:38]: I agree. No idea why it's showing that para as changed. It was there already.

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md (hunk 1 above)
Comment (lauracowen) [reply .github/ISSUE_TEMPLATE/blog_post_ga_release.md]: Done

**Pull request merged**